src/model/specific_path/LineType.java

```diff
@@ -0,0 +1,12 @@
+package model.specific_path;
+
+public enum LineType {
+
+	Line,
+	QuadricBezier,
+	SmoothBezier,
+	Curveto,
+	SmoothCurveto,
+	Arc
+	
+}
```

Comment:
I don't know...
I just found that for svg.
And if you find other things, I can add them, but for now I haven't found others